src/containers/CreateDoc/index.tsx

```diff
@@ -1,23 +1,55 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { connect, ConnectedProps } from 'react-redux';
 import VisibilityIcon from "@material-ui/icons/Visibility";
 import Button from "@material-ui/core/Button";
 import DeleteIcon from "@material-ui/icons/Delete";
 import GetAppIcon from "@material-ui/icons/GetApp";
 import Dropdown from "../../components/DocTypeDropdown";
 import DocumentContent from "../../components/DocumentContent";
+import Wizard from "../../components/Wizard";
 import documentTypes from "./config";
 import { createPdf } from "./pdf";
 import { setDocType as setDocTypeAction } from "../../actions/createDoc";
+import { 
+  setCurrentStep as setCurrentStepAction,
+  setDocSteps as setDocStepsAction 
+} from "../../actions/docSteps";
+import { Step } from '../../reducers/docStepsReducer';
 import { RootState } from "../../reducers/rootReducer";
 import createDocClasses from "./styles";
 import "./createDoc.css";
 
+const initialDocSteps = [
+  {
+    title: "Name",
```

Comment:
This component can be sent to independent files to be imported here. And we can suppor tmultiple documents